pages/manual/manual-gettingstarted-build.md

```diff
@@ -46,6 +46,16 @@ If you use a single-configuration generator, you can control the build mode usin
  Unless you specifically need to debug the application, we recommend setting the build mode to `RelWithDebInfo` for good performance, while still getting reasonable stacktraces for debugging and error reporting.
 " %}
 
+### Recommended setup (Optional)
+
+1. Build Type: Set the build mode to `RelWithDebInfo` ([See this guide for Visual Studio](https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2019))
```

Comment:
I think the note about debugability should stay... Stepping in RelWithDebInfo can be extremely confusing, especially if you are a novice...

Reply:
The note is still there after the (non-optional) windows build instructions

Reply:
Moved these instructions into the note above